core/controllers/admin.py

```diff
@@ -671,21 +670,30 @@ def _load_dummy_new_structures_data(self) -> None:
528.392 454.225 C 520.125 454.756, 514.144 457.825, 511.162 463.066 C 509.563 465.876, 507 467.069, 507 465.003 C 507 462.775, 499.622 455.989, 496.152 455.025 C 490.908 453.569, 481.059 453.835, 477.134 455.540 M 687.123 456.626 C 684.030 458.167, 680.954 460.641, 679.677 462.614 C 678.479 464.463, 677.163 465.981, 676.750 465.988 C 676.337 465.994, 676 463.525, 676 460.500 L 676 455 667.500 455 L 659 455 659 491 L 659 527 667.935 527 L 676.869 527 677.185 503.071 C 677.479 480.732, 677.637 478.958, 679.562 476.382 C 683.492 471.123, 686.673 469.500, 693.051 469.500 C 698.168 469.500, 699.324 469.869, 701.727 472.271 L 704.500 475.042 704.811 501.021 L 705.123 527 714.118 527 L 723.113 527 722.806 498.250 C 722.514 470.827, 722.398 469.310, 720.294 465.376 C 715.953 457.261, 709.718 454.016, 698.446 454.006 C 693.908 454.001, 691.073 454.657, 687.123 456.626 M 758.718 455.037 C 751.814 456.762, 740 464.111, 740 466.681 C 740 468.297, 746.190 477.957, 747.241 477.982 C 747.649 477.992, 749.992 476.466, 752.447 474.592 C 757.982 470.367, 762.837 468.714, 769.711 468.714 C 776.236 468.714, 779 470.784, 779 475.670 C 779 480.060, 775.194 482.449, 765.631 484.062 C 745.492 487.459, 738 493.635, 738 506.838 C 738 520.381, 746.101 528, 760.500 528 C 768.740 528, 772.668 526.374, 777.189 521.092 L 780.757 516.924 782.802 520.388 C 785.907 525.651, 789.063 527.250, 796.347 527.250 C 804.677 527.250, 806.783 526.097, 806.291 521.803 C 805.229 512.529, 805.114 512.256, 802.476 512.760 C 797.363 513.737, 797 512.366, 797 492.059 C 797 466.542, 795.401 461.793, 785 456.420 C 780.947 454.327, 764.936 453.484, 758.718 455.037 M 364.187 483.250 C 364.470 507.795, 364.748 512.023, 366.304 515.486 C 370.269 524.309, 376.521 527.944, 387.790 527.978 C 396.565 528.004, 401.119 526.016, 406.393 519.854 L 410 515.640 410 521.320 L 410 527 419 527 L 428 527 428 491 L 428 455 419.095 455 L 410.190 455 409.845 477.750 C 409.535 498.106, 409.289 500.914, 407.500 504.435 C 404.475 510.388, 400.948 512.500, 394.032 512.500 C 387.235 512.500, 384.570 510.710, 383.011 505.100 C 382.461 503.120, 382.009 491.038, 382.006 478.250 L 382 455 372.930 455 L 363.861 455 364.187 483.250 M 818 491 L 818 527 827 527 L 836 527 836 491 L 836 455 827 455 L 818 455 818 491 M 601.643 469.977 C 594.389 472.281, 591.053 478.895, 591.022 491.038 C 590.997 500.713, 593.272 506.758, 598.323 510.441 C 602.312 513.349, 611.747 513.324, 615.677 510.395 C 621.199 506.279, 622.500 502.581, 622.500 491 C 622.500 479.582, 621.178 475.690, 616 471.860 C 613.037 469.669, 605.659 468.701, 601.643 469.977 M 776.069 492.936 C 775.282 493.434, 771.761 494.385, 768.244 495.048 C 759.560 496.687, 756 499.701, 756 505.415 C 756 508.595, 756.599 510.214, 758.402 511.908 C 763.612 516.802, 773.440 514.953, 777.243 508.361 C 779.644 504.201, 780.877 491.994, 778.895 492.015 C 778.128 492.024, 776.856 492.438, 776.069 492.936" stroke="none" fill="#04645c" fill-rule="evenodd"/></svg>' # pylint: disable=line-too-long
```

Comment:
Please don't put a big chunk of text here. Read the necessary info from core/tests/data and do what's needed with it.

Most times, if you need to use a pylint-disable, something is wrong. Do not add pylint-disables in the code you commit. If you can't figure out how to avoid a pylint disable then please ask in GitHub discussions.

Reply:
Done